src/gui/CMakeLists.txt

```diff
@@ -364,6 +364,7 @@ SET(QGIS_GUI_SRCS
   qgsvscrollarea.cpp
   qgsdatasourcemanagerdialog.cpp
   qgsabstractdatasourcewidget.cpp
+  qgswidgetstatehelper_p.cpp
```

Comment:
I think they shouldn't be here. Private headers are not installed.

Reply:
hmmm... maybe for moc...

Reply:
Yeah doesn't work for me without.